docs/cli-gui.md

```diff
@@ -4,7 +4,7 @@ These instructions will take you through building an agent, starting an OEF Node
 
 ## Preliminaries
 
-Follow the Preliminaries and Installation instructions <a href="../quickstart" target=_blank>here</a>.
+Follow the Preliminaries and Installation instructions <a href="../quickstart">here</a>.
```

Comment:
@cyenyxe you and Josh added the target=_blank so it opens in a new window. @5A11 is removing it again. What do we want?